### DISCUSSION TITLE
Pyrad user manual

### DISCUSSION THREAD

**Discussion by wolfidan** [MeteoSwiss/pyrad#30] [General]
Ciao @jfigui 

It seems that that section _5.4.3	Add a recipe in conda-forge_ has to be done only once and no pyrad dev should have to repeat it?
If this is indeed true, I suggest to add a warning at the beg of that paragraph. Something like

**Note that this step has already been taken care of and should not be needed anymore. We leave it here for sake of exhaustive documentation.**

Let me know what you think

**Comment by jfigui** [comment]
Hi @wolfidan ,
I would keep it with the warning us you suggest. I think this documentation is the only one that I know where all the steps to manage a project are documented and I use it as a reference for other things.

**Reply by wolfidan** [reply]
Thanks, I did like that
